composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "studiometa/ui",
-  "version": "0.2.17",
+  "version": "0.2.19",
```

Comment:
**issue:** The previous version was `0.2.18`, you might need to re-fetch master before creating the hotfix 

**Small tip:** you can use `npm version patch` at the root of the project to update all version numbers in one command. Just make sure to disable tagging before running the command:

```sh
npm config -g set git-tag-version=false
npm version patch
```

Reply:
it seems 0.2.18 has only been pushed to `develop` not `master`. The hotfix branch does not exist on origin, so maybe it's easier if you can push @KeziahMoselle ?

Reply:
Sorry, it should be fixed now

Reply:
no worries 😄